app/src/main/java/io/homeassistant/companion/android/sensors/BatterySensorManager.kt

```diff
@@ -205,8 +205,7 @@ class BatterySensorManager : SensorManager {
     private fun getIsCharging(intent: Intent): Boolean {
         val status: Int = intent.getIntExtra(BatteryManager.EXTRA_STATUS, -1)
 
-        return status == BatteryManager.BATTERY_STATUS_CHARGING ||
-                status == BatteryManager.BATTERY_STATUS_FULL
+        return status == BatteryManager.BATTERY_STATUS_CHARGING
```

Comment:
Without the `FULL` check then when the battery is full `isCharging` will return `false` but that is not desired here as the device is still plugged in

Reply:
Agreed.